tasks/lzards-backup/schemas/input.json

```diff
@@ -12,20 +12,12 @@
         "type": "object",
         "required": [
           "files",
-          "granuleId",
-          "dataType",
-          "version"
+          "granuleId"
```

Comment:
https://github.com/nasa/cumulus/pull/3043/commits/3268eefcfb9ac32d2e9681495d37fa88beaff4ab

I added to `input.json` `dataType`, `version`, and `collectionId` but I don't see a way to indicate that the pair [dataType, version] are required together, or collectionId is required 🤔

Reply:
Ah so turns out I needed to revert this commit because the addition of those variables causes a CMA issue `EPIPE`? https://github.com/nasa/cumulus/pull/3043/commits/6fe25f5f26626a4c9fbd32480c370c289e092f24
🤔

Reply:
Alright, I've reverted that reversion 😅 https://github.com/nasa/cumulus/pull/3043/commits/e74a3cabd43f97d61f92c55c8981a05f43235c7c 
I couldn't figure how to indicate conditional parameters in JSON so I updated the description though I'm not sure it's sufficient

Reply:
Yeah, it seemed unlikely that there would be a slick way to handle this. I think updating the description and not explicitly making any of them required works fine 👍